lib/guardian_db.ex

```diff
@@ -1,11 +1,11 @@
-defmodule GuardianDb do
+defmodule GuardianDB do
```

Comment:
I'm not sure we want to rename the module @yordis

Reply:
@doomspork go to Slack channel and read the thread, basically most of the elixir guide lines out there recommend us to actually use uppercase for acronyms 
cc: @hassox